demos/git/README.md

```diff
@@ -0,0 +1,27 @@
+This example builds git trunk. It demonstrates the use of the 'home'
+plug to allow it access to the users home directory.
+
+After building and installing the snap, you will need to run the following
+command to grant it access to your home directory:
+
+```sh
+snap connect git:home ubuntu-core:home
+```
+
+Only access the home directory is granted. You can see this containment
```

Comment:
Fixed the grammar. I didn't explain about hidden files as I don't know what snappy considers hidden files - dotfiles seem to work just fine.